packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -49,7 +49,7 @@ import { VideoTile } from './VideoTile';
 import { useId } from '@fluentui/react-hooks';
 /* @conditional-compile-remove(local-camera-switcher) */
 import { LocalVideoCameraCycleButton, LocalVideoCameraCycleButtonProps } from './LocalVideoCameraButton';
-/* @conditional-compile-remove(call-with-chat-composite) @conditional-compile-remove(local-camera-switcher) */
```

Comment:
Removing all these conditional compilation directives themselves was unnecessary. Just moving the feature to `stablizedFeatures` has the same effect.

It's OK that you've done it now, but the PR can be simpler (lower risk) by not removing these at the same time.

Reply:
Having both the conditional compile for call-with-chat-composite and the local-camera-switcher was having issues when trying to keep only the local-camera-switcher in beta. 

When running `switch-flavor:stable && rush build -t communication-react`, it would complain about the apis for local-camera-switcher when both the conditional-compile-remove for call-with-chat-composite and local-camera-switcher were present.


I tried different combinations like:

```
/* @conditional-compile-remove(call-with-chat-composite) */
/* @conditional-compile-remove(local-camera-switcher) */

```


```
/* @conditional-compile-remove(call-with-chat-composite) @conditional-compile-remove(local-camera-switcher) */
```


```
/* @conditional-compile-remove(local-camera-switcher) */
/* @conditional-compile-remove(call-with-chat-composite) */
```

But all of them had issues for local-camera-switcher when placing call-with-chat-composite in stable until I removed the conditional-compile-remove(call-with-chat-composite).

Reply:
Reverted changes. Issue resolved in https://github.com/Azure/communication-ui-library/pull/1621